extensions/common/context.cc

```diff
@@ -428,6 +428,11 @@ void populateExtendedHTTPRequestInfo(RequestInfo* request_info) {
   getValue({"request", "url_path"}, &request_info->url_path);
   getValue({"request", "host"}, &request_info->url_host);
   getValue({"request", "scheme"}, &request_info->url_scheme);
+  std::string response_details;
+  getValue({"response", "code_details"}, &response_details);
```

Comment:
@kyessenov : It's not possible for both `connection_termination_details` and `response_code_details` to have values right as one is filled for TCP and other for HTTP

Reply:
nit: no need to stack allocate a string I think.

Reply:
will fix in the follow-up pr